clickhouse/client.cpp

```diff
@@ -352,22 +405,22 @@ bool Client::Impl::ReceivePacket(uint64_t* server_packet) {
     case ServerCodes::ProfileInfo: {
         Profile profile;
 
-        if (!WireFormat::ReadUInt64(&input_, &profile.rows)) {
+        if (!WireFormat::ReadUInt64(input_.get(), &profile.rows)) {
```

Comment:
This and all `.get()`s: won't it compile if you put just `input_`, etc.?

Reply:
@Enmk Have you tried not writing `.get()` in front of smart pointers when you pass them to OpenSSL calls? If it works, it will greatly unload the syntax and improve maintainability.

Reply:
This doesn't work and never supposed to work, since there are no implicit conversion from `std::unique_ptr<T>` to `T*`.